go-controller/observability-lib/model/network_event.go

```diff
@@ -23,39 +44,41 @@ type ACLEvent struct {
 func (e *ACLEvent) String() string {
 	var action string
 	switch e.Action {
-	case nbdb.ACLActionAllow, nbdb.ACLActionAllowRelated, nbdb.ACLActionAllowStateless:
+	case aclActionAllow, aclActionAllowRelated, aclActionAllowStateless:
 		action = "Allowed"
-	case nbdb.ACLActionDrop:
+	case aclActionDrop:
 		action = "Dropped"
-	case nbdb.ACLActionPass:
+	case aclActionPass:
 		action = "Delegated to network policy"
 	default:
 		action = "Action " + e.Action
 	}
 	var msg string
 	switch e.Actor {
-	case libovsdbops.AdminNetworkPolicyOwnerType:
+	case adminNetworkPolicyOwnerType:
 		msg = fmt.Sprintf("admin network policy %s, direction %s", e.Name, e.Direction)
-	case libovsdbops.BaselineAdminNetworkPolicyOwnerType:
+	case baselineAdminNetworkPolicyOwnerType:
 		msg = fmt.Sprintf("baseline admin network policy %s, direction %s", e.Name, e.Direction)
-	case libovsdbops.MulticastNamespaceOwnerType:
+	case multicastNamespaceOwnerType:
 		msg = fmt.Sprintf("multicast in namespace %s, direction %s", e.Namespace, e.Direction)
-	case libovsdbops.MulticastClusterOwnerType:
+	case multicastClusterOwnerType:
 		msg = fmt.Sprintf("cluster multicast policy, direction %s", e.Direction)
-	case libovsdbops.NetpolNodeOwnerType:
+	case netpolNodeOwnerType:
 		msg = fmt.Sprintf("default allow from local node policy, direction %s", e.Direction)
-	case libovsdbops.NetworkPolicyOwnerType:
+	case networkPolicyOwnerType:
 		if e.Namespace != "" {
 			msg = fmt.Sprintf("network policy %s in namespace %s, direction %s", e.Name, e.Namespace, e.Direction)
 		} else {
 			msg = fmt.Sprintf("network policy %s, direction %s", e.Name, e.Direction)
 		}
-	case libovsdbops.NetpolNamespaceOwnerType:
+	case netpolNamespaceOwnerType:
 		msg = fmt.Sprintf("network policies isolation in namespace %s, direction %s", e.Namespace, e.Direction)
-	case libovsdbops.EgressFirewallOwnerType:
+	case egressFirewallOwnerType:
 		msg = fmt.Sprintf("egress firewall in namespace %s", e.Namespace)
-	case libovsdbops.UDNIsolationOwnerType:
+	case udnIsolationOwnerType:
 		msg = fmt.Sprintf("UDN isolation of type %s", e.Name)
+	default:
+		msg = e.Actor
```

Comment:
now thinking about this again: do you think it makes sense to deplucate actor into the message vs just leaving msg empty? I think empty message gives a more clear signal that we don't know how to format messages from this actor and actor will be in the event struct anyway?

Reply:
done .. and squashed commits